x-pack/platform/packages/shared/kbn-elastic-assistant/impl/tour/const.ts

```diff
@@ -7,4 +7,5 @@
 
 export const NEW_FEATURES_TOUR_STORAGE_KEYS = {
   KNOWLEDGE_BASE: 'elasticAssistant.knowledgeBase.newFeaturesTour.v8.16',
+  ANONYMIZED_VALUES_AND_CITATIONS: 'elasticAssistant.anonymizedValuesAndCitationsTourCompleted',
```

Comment:
```suggestion
  ANONYMIZED_VALUES_AND_CITATIONS: 'elasticAssistant.anonymizedValuesAndCitationsTourCompleted.v8.18',
```

The tour storage key should include the version in case we implement an updated tour in future version